src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = [...table.rows];
+
+rows.forEach((row) => {
+  const cellContent = row.cells[1].innerHTML;
```

Comment:
In this line, you're assuming that the cell you're interested in is always at index 1. This might not always be the case. Consider finding a more robust way to reference the cell you're interested in. For example, you could use a data attribute or a specific class name.